docs/async_dispatch.rst

```diff
@@ -10,6 +10,8 @@ program:
 >>> import jax.numpy as jnp
 >>> from jax import random
 >>> x = random.uniform(random.PRNGKey(0), (1000, 1000))
+>>> # Printing the result or repr(result) or str(result) will block until the
```

Comment:
```suggestion
>>> # Printing the result (i.e. evaluating `repr(result)` or `str(result)`) will block until the
```

Maybe also tweak this to wrap at 80 character lines.

Reply:
Done